Wrong caret location on `@panic()` and `@errorReturnTrace()`

### Zig Version

0.10.0-dev.4192+c75e8f361

### Steps to Reproduce

Write some code that produces a runtime error or panic in a line with tabs, like so:
```zig
pub fn main() void {
	@panic("");
}
```
```zig
const std = @import("std");

pub fn someFunction() !void {
	_ = try std.math.mul(u32, std.math.maxInt(u32), 2);
}

pub fn main() void {
	 someFunction() catch {
		if(@errorReturnTrace()) |trace| {
			std.log.err("Trace: {}", .{trace});
		}
	};
}
```

### Expected Behavior

The caret should be at the location of the error, which is correctly handled for compiler errors(the tabs are converted to spaces while printing the source line which is good solution):
```zig
pub fn main() void {
	.
}
```
```
$ zig run test.zig
test.zig:2:2: error: expected statement, found '.'
 .
 ^
```

### Actual Behavior

The caret is pointing at the location where the error would be, if the tab was a space.

First example output:
```
thread 7137 panic: 
./test.zig:2:2: 0x210093 in main (test)
	@panic("");
 ^
```
Second example output:
```
error: Trace: 
[...]
./test.zig:4:6: 0x211ce9 in someFunction (test)
	_ = try std.math.mul(u32, std.math.maxInt(u32), 2);
     ^
```

--- a/lib/std/debug.zig
+++ b/lib/std/debug.zig
@@ -1128,7 +1128,10 @@ fn printLineFromFileAnyOs(out_stream: anytype, line_info: LineInfo) !void {
 
         for (slice) |byte| {
             if (line == line_info.line) {
-                try out_stream.writeByte(byte);
+                switch (byte) {
+                    '\t' => try out_stream.writeByte(' '),
+                    else => try out_stream.writeByte(byte),
+                }
                 if (byte == '\n') {
                     return;
                 }

